src/System.Private.CoreLib/shared/System/Collections/Generic/ArraySortHelper.cs

```diff
@@ -154,7 +152,8 @@ private static void SwapIfGreater(T[] keys, Comparison<T> comparer, int a, int b
             }
         }
 
-        private static void Swap(T[] a, int i, int j)
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
If this was found to important on Swap, I'm curious that it wasn't found to be important on SwapIfGreater?  Is that because SwapIfGreater already wasn't getting inlined?

Reply:
> because SwapIfGreater already wasn't getting inlined?

@stephentoub yes. And depending on input `SwapIfGreater` is not called nearly as often. Depends a lot on specific input data "median of three killer sequence" would probably change that statement. There are plenty of things that can be done to improve the performance of existing sorting code, as my previous PRs did. Although, .NET Core has also moved since so some things might no longer be needed.